### ISSUE TITLE
Can we check the detailed model parameters for Tabular Prediction

### ISSUE THREAD

**Issue by EternalConfession** [awslabs/autogluon#1215]
For example, can we check the e.g. learning rate, gamma, etc. for a single model.

Best,
Jianhui

**Comment by Innixma** [comment]
Yes,

with a trained predictor:

```
info = predictor.info()
info['model_info']['NeuralNetMXNet']['hyperparameters']
```

```
{'num_epochs': 500,
 'epochs_wo_improve': 20,
 'seed_value': None,
 'proc.embed_min_categories': 4,
 'proc.impute_strategy': 'median',
 'proc.max_category_levels': 100,
 'proc.skew_threshold': 0.99,
 'network_type': 'widedeep',
 'layers': None,
 'numeric_embed_dim': None,
 'activation': 'relu',
 'max_layer_width': 2056,
 'embedding_size_factor': 1.0,
 'embed_exponent': 0.56,
 'max_embedding_dim': 100,
 'y_range': None,
 'y_range_extend': 0.05,
 'use_batchnorm': True,
 'dropout_prob': 0.1,
 'batch_size': 512,
 'loss_function': None,
 'optimizer': 'adam',
 'learning_rate': 0.0003,
 'weight_decay': 1e-06,
 'clip_gradient': 100.0,
 'momentum': 0.9,
 'lr_scheduler': None,
 'base_lr': 3e-05,
 'target_lr': 1.0,
 'lr_decay': 0.1,
 'warmup_epochs': 10,
 'use_ngram_features': False}
```

Note that these are hyperparameters that are used in AutoGluon, you'll notice some things like `'layers': None`. This has special meaning in AG that involves dynamic default layers. AG will generally not return the actual layers used in this case, but instead the hyperparameters used at the start of training. Therefore, these hyperparameters are not necessarily transferrable directly outside of AG, but rather can be used to specify a new model's hyperparameters.

**Comment by EternalConfession** [comment]
Thanks for your reply.

Best,
Jianhui
